src/StructuredLogger/Analyzers/DoubleWritesAnalyzer.cs

```diff
@@ -36,7 +36,27 @@ public void AppendDoubleWritesFolder(Build build)
             }
         }
 
-        public void AnalyzeFileCopies(CopyTask copyTask)

```

Comment:
This does change an API that was previously public. I'm unsure how critical this is or isn't for this component. Can add it back if needed.